tests/Functional/src/com/microsoft/aad/adal/test/AuthenticationContextTest.java

```diff
@@ -1364,6 +1367,98 @@ public void testRefreshTokenRequestWithFamilyIdFailed() throws InterruptedExcept
         clearCache(context);
     }
     
+    public void testMultipleFamilyItemsInCache_LatestOneWillBePicked() throws NoSuchAlgorithmException, NoSuchPaddingException,       
+             NoSuchFieldException, IllegalAccessException, JSONException, InterruptedException {
+         FileMockContext mockContext = new FileMockContext(getContext());
+         ITokenCacheStore mockCache = new DefaultTokenCacheStore(getContext());
+         
+         // add more items in the token cache
+         // token cache item 1 with FoCI flag, set token updated time to be 1 month later
+         final TokenCacheItem tokenItem1 = createTokenCacheItemWithFoCIFlag("authority1", "clientid1", "accessToken1", "refreshToken1", 
+                 TEST_IDTOKEN_USERID, TEST_IDTOKEN_UPN);
+         final Calendar tokenUpdatedTime1 = new GregorianCalendar();
+         tokenUpdatedTime1.add(Calendar.MONTH, 1);
+         tokenItem1.setTokenUpdateTime(tokenUpdatedTime1.getTime());
+         addTokenCacheItemInCache(tokenItem1, mockCache);
```

Comment:
why do you specifically set time for tokens? When you create token it will generate time and next created token will be "younger" than previous one